Development/nmos/test/video_jxsv_test.cpp

```diff
@@ -56,3 +57,128 @@ a=fmtp:112 packetmode=0; profile=High444.12; level=1k-1; sublevel=Sublev3bpp; de
         BST_CHECK_EQUAL(expected_line, actual_line);
     } while (!expected.fail() && !actual.fail());
 }
+
+////////////////////////////////////////////////////////////////////////////////////////////
+BST_TEST_CASE(testSdpParametersVideoJpegXs)
+{
+    std::pair<nmos::sdp_parameters, nmos::video_jxsv_parameters> example{
+        {
+            U("example"),
+            sdp::media_types::video,
+            {
+                112,
+                U("jxsv"),
+                90000
+            },
+            {
+                { U("packetmode"), U("0") },
+                { U("profile"), U("High444.12") },
+                { U("level"), U("1k-1") },
+                { U("sublevel"), U("Sublev3bpp") },
+                { U("sampling"), U("YCbCr-4:2:2") },
+                { U("width"), U("1280") },
+                { U("height"), U("720") },
+                { U("exactframerate"), U("60000/1001") },
+                { U("depth"), U("10") },
+                { U("colorimetry"), U("BT709") },
+                { U("TCS"), U("SDR") },
+                { U("RANGE"), U("FULL") },
+                { U("SSN"), U("ST2110-22:2019") },
+                { U("TP"), U("2110TPN") }
+            },
+            116000
+        },
+        {
+            sdp::video_jxsv::packetization_mode::codestream,
+            sdp::video_jxsv::transmission_mode::sequential,
+            sdp::video_jxsv::profiles::High444_12,
+            sdp::video_jxsv::levels::Level1k_1,
+            sdp::video_jxsv::sublevels::Sublev3bpp,
+            sdp::samplings::YCbCr_4_2_2,
+            10,
+            1280,
+            720,
+            nmos::rates::rate59_94,
+            false,
+            false,
+            sdp::transfer_characteristic_systems::SDR,
+            sdp::colorimetries::BT709,
+            sdp::ranges::FULL,
+            sdp::smpte_standard_numbers::ST2110_22_2019,
+            sdp::type_parameters::type_N,
+            {},
+            {},
+            {},
+            {},
+            {},
+            116000
+        }
+    };
+
+    std::pair<nmos::sdp_parameters, nmos::video_jxsv_parameters> wacky{
+        {
+            U("example"),
```

Comment:
```suggestion
            U("wacky"),
```

Can we make this example... wackier...?

If we include some of the lesser used values for each of the parameters, we demonstrate we can roundtrip those, and at least that we don't hard code any of the "standard" example ones.